### ISSUE TITLE
Loss变化很不规律

### ISSUE THREAD

**Issue by horizon-z40** [ZJULearning/pixel_link#19]
INFO:tensorflow:global step 368468: loss = 1.0245 (0.416 sec/step)
INFO:tensorflow:global step 368469: loss = 1.0627 (0.404 sec/step)
INFO:tensorflow:global step 368470: loss = 1.0460 (0.399 sec/step)
INFO:tensorflow:global step 368471: loss = 0.8222 (0.410 sec/step)
INFO:tensorflow:global step 368472: loss = 1.0190 (0.408 sec/step)
INFO:tensorflow:global step 368473: loss = 0.7397 (0.410 sec/step)
INFO:tensorflow:global step 368474: loss = 0.7372 (0.412 sec/step)
INFO:tensorflow:global step 368475: loss = 0.7329 (0.412 sec/step)
INFO:tensorflow:global step 368476: loss = 1.0552 (0.420 sec/step)
INFO:tensorflow:global step 368477: loss = 0.7700 (0.398 sec/step)
INFO:tensorflow:global step 368478: loss = 0.6954 (0.418 sec/step)
INFO:tensorflow:global step 368479: loss = 0.7515 (0.397 sec/step)
INFO:tensorflow:global step 368480: loss = 1.2682 (0.411 sec/step)
INFO:tensorflow:global step 368481: loss = 0.9167 (0.398 sec/step)
INFO:tensorflow:global step 368482: loss = 1.2476 (0.416 sec/step)
INFO:tensorflow:global step 368483: loss = 0.9134 (0.419 sec/step)
INFO:tensorflow:global step 368484: loss = 0.6969 (0.414 sec/step)
INFO:tensorflow:global step 368485: loss = 0.8940 (0.407 sec/step)
INFO:tensorflow:global step 368486: loss = 0.9226 (0.394 sec/step)
INFO:tensorflow:global step 368487: loss = 1.0456 (0.420 sec/step)
INFO:tensorflow:global step 368488: loss = 0.6929 (0.406 sec/step)
INFO:tensorflow:global step 368489: loss = 0.9335 (0.411 sec/step)
INFO:tensorflow:global step 368490: loss = 0.8754 (0.410 sec/step)
INFO:tensorflow:global step 368491: loss = 0.9203 (0.411 sec/step)
INFO:tensorflow:global step 368492: loss = 1.0077 (0.420 sec/step)
INFO:tensorflow:Recording summary at step 368492.
INFO:tensorflow:global step 368493: loss = 0.8308 (0.588 sec/step)
INFO:tensorflow:global step 368494: loss = 0.6966 (0.422 sec/step)
INFO:tensorflow:global step 368495: loss = 0.9326 (0.411 sec/step)
INFO:tensorflow:global step 368496: loss = 0.7347 (0.421 sec/step)
INFO:tensorflow:global step 368497: loss = 0.8080 (0.419 sec/step)
INFO:tensorflow:global step 368498: loss = 0.8667 (0.409 sec/step)
INFO:tensorflow:global step 368499: loss = 1.2679 (0.407 sec/step)
INFO:tensorflow:global step 368500: loss = 0.8499 (0.418 sec/step)

**Comment by 741077510** [comment]
+1

**Comment by BowieHsu** [comment]
那是因为这是每次迭代都输出的loss，你调成每10次迭代输出就感觉规律了

**Comment by LinnaWang76** [comment]
用作者的源码训练自己的数据，然后gpu没使用上，cpu也使用率不高，基本90 sec/step，作为小白想问下怎么开启gpu训练

**Comment by BowieHsu** [comment]
@LinnaWang76 检查一下你的tensorflow是不是用的gpu版本？

**Comment by LinnaWang76** [comment]
@BowieHsu 谢谢！

**Comment by LinnaWang76** [comment]
@BowieHsu 想请问下 我怎么评估网络，训练的时候只显示了loss，而且 按照作者的readme ，训练icdar数据集，迭代到30000步，loss到1.7左右降不下来，然后test_any ，icdar上面的测试集，什么都检测不到。

**Comment by BowieHsu** [comment]
@LinnaWang76 如果要评估网络的话，你需要把icdar的offline acc\recall计算的那段python代码拿来复用一下，这样能算出在指定数据集上的准确率和召回率。

**Comment by LinnaWang76** [comment]
@BowieHsu 谢谢大佬的回复，这个是在训练好模型之后进行的模型评估，我主要是想在训练的时候，不仅仅显示golab_step和loss，同时显示acc和recall，但是这里代码用的是slim.learning.train的方法，该怎么输出计算的acc和recall。。小白，请见怪，找了一天也没找到怎么输出，只知道这个loss输出都是封装在slim里面的
